Resources/Locale/en-US/escape-menu/ui/options-menu.ftl

```diff
@@ -113,7 +113,7 @@ ui-options-function-move-up = Move Up
 ui-options-function-move-left = Move Left
 ui-options-function-move-down = Move Down
 ui-options-function-move-right = Move Right
-ui-options-function-walk = Walk
+ui-options-function-walk = Run
```

Comment:
misleading fluent key

Reply:
Don't know why this is the engine. I can't change this.

![image](https://github.com/Simple-Station/Einstein-Engines/assets/77995199/69179104-14b4-4f49-86a2-d7c93be6bfe1)

![image](https://github.com/Simple-Station/Einstein-Engines/assets/77995199/4610755c-5537-4c02-83f3-6efa95eb5edd)